wiki/Game_Modifiers/en.md

```diff
@@ -78,9 +78,9 @@
 
 Game modifiers (mods) are optional modifications to a beatmap's elements and settings that the player can enable at will. A mod effect can range from simple gameplay changes like changing the beatmap's settings to automation such as allowing _osu!_ to do a perfect replay (Auto), on a condition of possible Score Multiplier changes to making the play unrankable. When multiple mods are enabled, the Score Multiplier is the product of the enabled mods' Score Multiplier. For example, when Hidden and Flashlight are applied at the same time, the Score Multiplier will be 1.19x (1.06 * 1.12 = 1.1872; 1.19 by rounding up to nearest hundredths).
 
-Shortcut keys to enable the mods can be changed in the Options under the Keyboard header inside `Change keyboard bindings` with a header called "Mod Select". The usage of the shortcut keys will only work when the mods screen is shown.
+Shortcut keys to enable the mods can be changed in the Options under the Keyboard header inside `Change keyboard bindings` with a header called "Mod Select". The usage of the shortcut keys will only work when the mod selection screen is shown.
```

Comment:
* remove 'the' before mods
* 'in the options under the keyboard inside...' is a whole lot of prepositional phrases, suggest rewording this